server/controllers/wazuh-reporting.ts

```diff
@@ -261,19 +238,37 @@ export class WazuhReportingCtrl {
           }
         }
       }
-      printer.addSimpleTable({
-        columns: [
-          { id: 'id', label: 'ID' },
-          { id: 'name', label: 'Name' },
-          { id: 'ip', label: 'IP' },
-          { id: 'version', label: 'Version' },
-          { id: 'manager', label: 'Manager' },
-          { id: 'os', label: 'OS' },
-          { id: 'dateAdd', label: 'Registration date' },
-          { id: 'lastKeepAlive', label: 'Last keep alive' },
-        ],
-        items: agentRows,
-      });
+
+      if(agentsData.length){
+        // Print a table with agent/s information
+        printer.addSimpleTable({
+          columns: [
+            { id: 'id', label: 'ID' },
+            { id: 'name', label: 'Name' },
+            { id: 'ip', label: 'IP' },
+            { id: 'version', label: 'Version' },
+            { id: 'manager', label: 'Manager' },
+            { id: 'os', label: 'OS' },
+            { id: 'dateAdd', label: 'Registration date' },
+            { id: 'lastKeepAlive', label: 'Last keep alive' },
+          ],
+          items: agentsData.map((agent) => {
+            return {
+              ...agent,
+              os: (agent.os && agent.os.name && agent.os.version) ? `${agent.os.name} ${agent.os.version}` : '',
+              lastKeepAlive: moment(agent.lastKeepAlive).format(dateFormat),
+              dateAdd: moment(agent.dateAdd).format(dateFormat)
+            }
+          }),
+        });
+      }else if(!agentsData.length && groupID){
+        // For group reports when there is no agents in the group
+        printer.addContent({
+          text: 'There are still no agents in this group.',
```

Comment:
A more appropriate sentence would be: `There are no agents in this group yet` or simply `There are no agents in this group`

Reply:
This message was included in the PDF report. The message was not changed. https://github.com/wazuh/wazuh-kibana-app/pull/4350/files#diff-e344f646f1844581c2ba5159d750a8f075c476e56350e3bcc6018f87136dd973L142

But I will take your suggestion. Thank you!

Reply:
Changed.